test/e2e/common/node_lease.go

```diff
@@ -68,12 +78,72 @@ var _ = framework.KubeDescribe("[Feature:NodeLease][NodeAlphaFeature:NodeLease]"
 				}
 				return nil
 			}, time.Duration(*lease.Spec.LeaseDurationSeconds)*time.Second,
-				time.Duration(*lease.Spec.LeaseDurationSeconds/3)*time.Second)
+				time.Duration(*lease.Spec.LeaseDurationSeconds/4)*time.Second)
+		})
+
+		It("the kubelet should report node status infrequently", func() {
+			By("wait until node is ready")
+			framework.WaitForNodeToBeReady(f.ClientSet, nodeName, 5*time.Minute)
+
+			By("wait until there is node lease")
+			var err error
+			var lease *coordv1beta1.Lease
+			Eventually(func() error {
+				lease, err = f.ClientSet.CoordinationV1beta1().Leases(corev1.NamespaceNodeLease).Get(nodeName, metav1.GetOptions{})
+				if err != nil {
+					return err
+				}
+				return nil
+			}, 5*time.Minute, 5*time.Second).Should(BeNil())
+			// check basic expectations for the lease
+			Expect(expectLease(lease, nodeName)).To(BeNil())
+			leaseDuration := time.Duration(*lease.Spec.LeaseDurationSeconds) * time.Second
+
+			By("verify NodeStatus report period is longer than lease duration")
+			// NodeStatus is reported from node to master when there is some change or
+			// enough time has passed. So for here, keep checking the time diff
+			// between 2 NodeStatus report, until it is longer than lease duration.
```

Comment:
Checking for 2*lease_duration may be better since kubelet could just be a bit slow to send out the update

Reply:
Discussed offline. Updated the comments to make it more clear. 

Also added more comments on checking node status.